Dockerfile.builder

```diff
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile:1
+ARG PYTHON_VERSION=3.11
```

Comment:
Add comment for viewer:
This is the base image that installs the build environment and requirements. You depend on this builder image in customised images for dev, production, acceptance or your own projects.